### ISSUE TITLE
Is kraken2 suitable for tNGS data? 

### ISSUE THREAD

**Issue by zhuying412** [DerrickWood/kraken2#896]
I know that Kraken2 is commonly used for metagenomic data analysis and species classification. I wonder if it is suitable for tNGS (targeted Next-Generation Sequencing) data, such as data obtained from ultra-multiplex PCR capture or probe capture?

**Comment by jmonroynieto** [comment]
Yes you can use it with manual definitions. I do this regularly but have never tried to do it with  small targets. Because kraken is designed as a taxonomic classifier, I also would recommend that you be very mindful of your objective and the underlying differentiators.

Are you targeting a barcoding sequence? Are you using a set of unique genes for a taxon? Are you wanting to do presence/absence detection for a multiplexed assay?
